ignite/distributed/utils.py

```diff
@@ -3,6 +3,12 @@
 from typing import Any, Callable, List, Mapping, Optional, Tuple, Union
 
 import torch
+import torch.distributed as dist
+from horovod.common.process_sets import ProcessSet
```

Comment:
To add `def new_group` it is required to import `horovod.common.process_sets.ProcessSet` and `ProcessGroup`.
But it fails when import. How can I deal with this?